### PR TITLE
Update TensorboardGenerativeModelImageSampler  args

### LINKED ISSUE #489: Add nrow and normalize args to TensorboardGenerativeModelImageSampler
## 🚀 Feature
Add `nrow` and `normalize` args to [`TensorboardGenerativeModelImageSampler`](https://github.com/PyTorchLightning/PyTorch-Lightning-Bolts/blob/master/pl_bolts/callbacks/vision/image_generation.py#L12-L53)

### Motivation

At the moment, the image grid that is added to Tensorboard is created with `grid = torchvision.utils.make_grid(images)`. It would be nice to have control over the `nrow` and `normalize` args of `make_grid`, e.g. through `grid = torchvision.utils.make_grid(images, nrow=self.nrow, normalize=self.normalize)`.

### Pitch

Change `TensorboardGenerativeModelImageSampler`'s init from
```python
    def __init__(self, num_samples: int = 3):
        super().__init__()
        self.num_samples = num_samples
```

to

```python
    def __init__(self, num_samples: int = 3, nrow: int = 1, normalize: bool = True):
        super().__init__()
        self.num_samples = num_samples
        self.nrow = nrow
        self.normalize = normalize
```

and `grid = torchvision.utils.make_grid(images)` to `grid = torchvision.utils.make_grid(images, nrow=self.nrow, normalize=self.normalize)`.

### Alternatives

- Add args for all of `make_grid`'s args with sensible default values
- Create a general image logging callback that can deal with all loggers and not just Tensorboard

### PR DESCRIPTION
**Pull request by chris-clem**
## What does this PR do?
Updates TensorboardGenerativeModelImageSampler args as discussed in #489.

Fixes #489

## Before submitting
- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? [not needed for typos/docs]
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/CHANGELOG.md)?

## PR review
 - [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: pl_bolts/callbacks/vision/image_generation.py [hunk 1]
```diff
@@ -30,9 +32,38 @@ class TensorboardGenerativeModelImageSampler(Callback):
...
+                The final grid size is ``(B / nrow, nrow)``. Default: ``8``.
+            padding: amount of padding. Default: ``2``.
+            normalize: If True, shift the image to the range (0, 1),
+                by the min and max values specified by :attr:`range`. Default: ``False``.
+            norm_range: tuple (min, max) where min and max are numbers,
+                then these numbers are used to normalize the image. By default, min and max
+                are computed from the tensor.
+            scale_each: If ``True``, scale each image in the batch of
+                images separately rather than the (min, max) over all images. Default: ``False``.
+            pad_value: Value for the padded pixels. Default: ``0``.
```
Comment (akihironitta) [inline pl_bolts/callbacks/vision/image_generation.py:57]: Could you add `num_samples` to the docstring?

File: pl_bolts/callbacks/vision/image_generation.py [hunk 2]
```diff
@@ -30,9 +32,38 @@ class TensorboardGenerativeModelImageSampler(Callback):
...
+        norm_range: Optional[Tuple[int, int]] = None,
+        scale_each: bool = False,
+        pad_value: int = 0,
+    ) -> None:
+        """
+        Args:
+            nrow: Number of images displayed in each row of the grid.
+                The final grid size is ``(B / nrow, nrow)``. Default: ``8``.
+            padding: amount of padding. Default: ``2``.
+            normalize: If True, shift the image to the range (0, 1),
```
Comment (akihironitta) [inline pl_bolts/callbacks/vision/image_generation.py:50]:
```suggestion
            normalize: If ``True``, shift the image to the range (0, 1),
```

**Review by akihironitta — CHANGES_REQUESTED** [review]
@chris-clem Thank you for your swift action! Would you mind having a look at the below suggestions?

**Comment by akihironitta** [comment]
@Borda @ananyahjha93 Should we include the tests for this class in this PR? or later in another PR?

**Review by Borda — APPROVED** [review]

**Pull request merged**